build_tools/packaging/python/templates/rocm-sdk-core/setup.py

```diff
@@ -63,9 +64,18 @@ def import_dist_info():
             "roc-obj=rocm_sdk_core._cli:roc_obj",
             "roc-obj-extract=rocm_sdk_core._cli:roc_obj_extract",
             "roc-obj-ls=rocm_sdk_core._cli:roc_obj_ls",
```

Comment:
What are these `roc_obj` console scripts? I don't see matching functions in https://github.com/ROCm/TheRock/blob/main/build_tools/packaging/python/templates/rocm-sdk-core/src/rocm_sdk_core/_cli.py.